datasets/minmax.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/zsh
+
+# This is a very handwavy script to allow to convert the user input values as coming
+# from the measurements to the values used in the nodes to train.
```

Comment:
maybe just add a line saying that it generated the data used in [webapp/src/app/query-runner/query-runner.component.ts](https://github.com/c4dt/service-spindle/pull/76/files#diff-22ee2e3457ded6cdd7bd44de8dd8d2037fc77c9a159c84b9ac0690d83aff62d4)